### PR TITLE
Part of #17670: Replace Typography with Text component for 11 files.

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a part of the issue #17670 Replace Typography with Text component for 11 files:

1.  ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js
2.  ui/components/app/recovery-phrase-reminder/recovery-phrase-reminder.js
3.  ui/components/app/security-provider-banner-message/security-provider-banner-message.js
4.  ui/components/ui/deprecated-test-networks/deprecated-test-networks.js
5.  ui/components/ui/form-field/form-field.js
6.  ui/components/ui/form-field/form-field.stories.js
7.  ui/components/ui/logo/logo.stories.js
8.  ui/components/ui/metafox-logo/metafox-logo.stories.js
9.  ui/components/ui/new-network-info/new-network-info.js
10. ui/components/ui/nft-info/nft-info.js
11.ui\components\ui\radio-group\radio-group.component.js

Also `boxProps` have been removed in `Text` element in some files.

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by kremalicious** [comment]
hey there, as an occasional contributor also working on this `Text` replacement just seeing that you open up a lot of PRs ignoring reviewer requests on how to do that. As a reviewer myself I know how much work this puts on reviewers, and you are taking away review time from many other PRs.

So, please, follow the original issue instructions, especially:

- 1 component per PR (this one touches 11, why??)
- screenshots of the changes, even if there are no visual changes (that's the point). Screenshots of actual UI, NOT screenshots of your code editor.

Until every PR you have opened follows this I would advise you stop opening up new PRs

**Comment by PrgrmrHarshShukla** [comment]
> hey there, as an occasional contributor also working on this `Text` replacement just seeing that you open up a lot of PRs ignoring reviewer requests on how to do that. As a reviewer myself I know how much work this puts on reviewers, and you are taking away review time from many other PRs.
> 
> So, please, follow the original issue instructions, especially:
> 
> * 1 component per PR (this one touches 11, why??)
> * screenshots of the changes, even if there are no visual changes (that's the point). Screenshots of actual UI, NOT screenshots of your code editor.
> 
> Until every PR you have opened follows this I would advise you stop opening up new PRs


Thanks a lot @kremalicious for the advices.❤️
I will definitely follow them.

I am really sorry for the blunders 
& really thankful to the reviewers 
@georgewrmarshall , @brad-decker , @DDDDDanica , @pedronfigueiredo , 
and others for bearing with me.

I assure everyone that the blunders will not be repeated 
&
I will go and fix them in the currently open PRs now.

I have just started doing open source contributions & thankful to everyone for bearing with me.🙏🏼


Thank you again.
Regards.

**Comment by brad-decker** [comment]
@PrgrmrHarshShukla -- don't fret. We opened up these issues to the public fully aware that it was going to take time for us to review! The only mistakes are the one you repeat while expecting different results, blunders are life's big teaching moments. I have failed a thousand times, and hope to keep failing, because if I am always successful I am not growing. 

@kremalicious, thank you for pushing for quality in these Pull Requests and I know that it can be frustrating while touching the same issue to juggle between multiple authors who are all chipping away at the same iceberg. I know the spirit of your comment is in an effort to help @PrgrmrHarshShukla, but just keep in mind that we're all learning and working through these issues as a team. I'd only ask that in future messages to fellow contributors you make **suggestions** "You could probably get these reviewed faster if you were to align your pull request with the issue, which stated one pull request per file, this one has eleven and it may mean it doesn't get reviewed as quickly" versus the language you chose which felt more derisive (even if unintended). 

We make MetaMask better by working together, and helping each other grow.

**Review by brad-decker — APPROVED** [review]

File: ui/components/ui/form-field/form-field.js [hunk 1]
```diff
@@ -132,40 +132,44 @@ export default function FormField({
...
+            variant={TextVariant.bodyXs}
+            as="h6"
             className="form-field__password-strength-text"
           >
             {passwordStrengthText}
-          </Typography>
+          </Text>
         )}
       </Box>
     </div>
```
Comment (georgewrmarshall) [inline ui/components/ui/form-field/form-field.js:175]: Adding screenshots for FormField

### Before
Screenshot 2023-04-29 at 8 43 10 AM

### After
Screenshot 2023-04-29 at 8 41 40 AM

**Review by georgewrmarshall — APPROVED** [review]
Although the code looks good I'm hesitant to approve this PR because it doesn't meet the acceptance criteria of adding Before/After screenshots to the PR description or to the code files. It also contains more than a single file which has made it a lot more time consuming to review. I really appreciate all your contributions but without meeting the acceptance criteria which really helps with reviews I will have to close them. I will also try to make sure this is clear in the issues that I am creating.

I've added the before / after screenshots as an example for the FormField. You can use storybook locally by running 
```
yarn storybook
```

and checking against the same story in the `develop` branch of storybook hosted here
https://metamask.github.io/metamask-storybook/?path=/story/getting-started-introduction--page

For any future PRs please complete the acceptance criteria or they may be closed. Thanks again!

**Pull request merged**